server/src/main/java/com/broadcom/lsp/cobol/core/visitor/CobolVisitor.java

```diff
@@ -436,12 +444,21 @@ private void addUsage(SubContext<?> langContext, String name, @NonNull Location
     langContext.addUsage(name.toUpperCase(), location);
   }
 
+  private void addParagraphUsage(String name, @NonNull Locality locality) {
+    paragraphs.addUsage(name.toUpperCase(), locality.toLocation());
```

Comment:
The `name` is already in UpperCase. It's done in `visitParagraphNameUsage` method.

Reply:
Thanks, fixed.